lingetic-nextjs-frontend/app/languages/learn/[language]/page.tsx

```
@@ -72,16 +73,30 @@ export default function LearnPage() {
+  const Component = questionTypeToComponentMap[question.questionType];
+  return <Component question={question} onAnswerSubmit={onAnswerSubmit} />;
+};
+
+const validateQuestionOrDie = (question: any) => {
+  assert(question != null, "question is null or undefined");
+  assert(
+    question.questionType != null,
+    "question.questionType is null or undefined"
+  );
+  assert(
+    questionTypeToComponentMap.hasOwnProperty(question.questionType),
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)